src/main/scala/vectorpipe/osm/internal/util/Caching.scala

```diff
@@ -6,7 +6,7 @@ import org.apache.spark.sql._
 import java.io.File
 
 
-trait Caching {
+trait Caching extends Serializable {
```

Comment:
Is this still used?

Reply:
Just in the `OSMReader`, but I think that can probably be taken out